tests/py/test_www_team_distributing.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from gratipay.testing import Harness
+
+
+class Tests(Harness):
+
+    def test_distributing_redirects_when_no_money_is_available(self):
+        self.make_team()
+        assert self.client.GxT('/TheEnterprise/distributing/').code == 302
+
+    def test_distributing_doesnt_redirect_when_money_is_available(self):
+        self.make_team()
+        self.db.run("UPDATE teams SET available=537")
+        assert self.client.GET('/TheEnterprise/distributing/', raise_immediately=False).code == 500
```

Comment:
It's a 500 that no user would've hit, since it's behind a flag (`available`) which no Team has the power to set themselves—making this PR deployable while also keeping it reasonably sized. However, this PR landed on master [together](https://github.com/gratipay/gratipay.com/issues/3994#issuecomment-232237350) with the next one (#4073), which includes a fix for the 500 (17b9bd37de4a9b088b8d4cd4732943cc9bf28d21). Now, #4073 *also* introduces a test for [_another_ 500](https://github.com/gratipay/gratipay.com/commit/4cfd33f33c51f81a9defa06a4513b5a4b978f1cf#diff-bdbc0e30f418dad4e477decfc410f9d1R35), which is [still](https://github.com/gratipay/gratipay.com/blob/1976/tests/py/test_www_team_distributing.py#L32-L35) on `master`. Again, my [thinking](https://github.com/gratipay/gratipay.com/issues/3994#issuecomment-229949952) is that deploying broken code behind the `available` flag allows us to keep chipping away with manageable PRs instead of batching up until everything is "perfect." You okay with that?